### DISCUSSION TITLE
Pinecil v2 ble

### DISCUSSION THREAD

**Discussion by mikerocklewitz** [Ralim/IronOS#1449] [Q&A]
Its mentioned in the pinecil v2 documentation that ble will be implemented in the future.  Im just curious. What is the plan for ble? How is it going to be used?

**Comment by Ralim** [comment]
There is some existing thoughts on here,but TL;DR is to expose all settings and also the live status.

This opens to floor for apps or integrations

**Comment by builder555** [comment]
I remember seeing a BLE branch at some point, is it gone now? I can't seem to enable BLE (it's not in the advanced settings), and I don't see the device when scanning with [bleak](https://pypi.org/project/bleak/). I flashed the latest firmware (2.20.14DAF70). @Ralim 

Update: flashed [this](https://github.com/Ralim/IronOS/actions/runs/4088413978#artifacts) artifact, now BLE shows up! 👍

**Reply by Ralim** [reply]
BLE is now part of `dev`. It's been merged in.
